app/messages.js

```diff
@@ -1,16 +1,13 @@
-define(['jquery', 'app/timer', 'app/string'], function ($, timer, string) {
+define(['jquery', 'app/lang', 'app/timer', 'app/string'], function ($, lang, timer, string) {
   /* Function show for displaying normal, danger and warning messages.
   * The message is included as html and a class with the corresponding type is
   * added. Within the switch statement the icon is added.
   */
-  
-  const THOUSAND = 1000;
 
   return {
     show: function(message, type, countdown) {
-      countdown /= THOUSAND;
       if (countdown > 0) {
-        $("#show .message").html(message + "<br> <span class=\"countdownnotice\">Press enter to continue or wait <span class=\"countdown\" data-seconds=\"" + countdown + "\"></span> second" + string.pluralIfAppropriate(countdown) + ".</span>");
+        $("#show .message").html(message + "<br> <span class=\"countdownnotice\">" + lang("message_enterorwait", "<span class=\"countdown\" data-seconds=\"" + countdown + "\"></span>"));
```

Comment:
@sdejong You have added the placeholder string for the internationalisation, but you still add html in your javascript. What I meant, is that you should have the html in your view and replace the text you want to change by changing spans in the html, not by building html in here.